<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_addprog</class>
 <widget class="QWidget" name="window_addprog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selectioner la tâche à programmer</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_program">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Programmation d'une tâche</string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="t_titre"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="t_date"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="b_ok">
      <property name="text">
       <string>Programmer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Programmation d'un évènement</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Titre</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="t_titreEv"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateTimeEdit" name="t_dateEv"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="b_progEv">
      <property name="text">
       <string>Programmer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
